pynet/datasets/biomede.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# NSAp - Copyright (C) CEA, 2019
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+# @Author: Zhou Kai
```

Comment:
Those three lines point to an implementation of the NvNet but are not linked to the fetcher itself.